launch.html

```diff
@@ -0,0 +1,15 @@
+<html lang="en">
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>SMART on FHIR Starter APP</title>

```

Comment:
Let's standardize on this name.  `SMART on FHIR Starter App` ?  `SMART Starter App` ?